lib/task-data/base-tasks/ucs-service-profile-discovery.js

```diff
@@ -0,0 +1,17 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+module.exports = {
+    friendlyName: 'UCS Service Profile Discovery Job',
+    injectableName: 'Task.Base.UCS.Service.Profile.Discovery',
+    runJob: 'Job.Ucs.Service.Profile.Discovery',
+    requiredOptions: [
+        'uri'
```

Comment:
@yyscamper this will be refactored as part of https://rackhd.atlassian.net/browse/RAC-4389